Xamarin.Forms.Core/Xamarin.Forms.Core.csproj

```diff
@@ -89,10 +89,9 @@
     <Compile Include="EnumerableExtensions.cs" />
     <Compile Include="PlatformConfiguration\AndroidSpecific\AppCompat\Application.cs" />
     <Compile Include="PlatformConfiguration\AndroidSpecific\Application.cs" />
-    <Compile Include="PlatformConfiguration\AndroidSpecific\TabbedPage.cs" />
```

Comment:
Sorry. I had troubles with rebase & merge recently that affected this and another PR. It should be fixed.